src/libs/Violations/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as ViolationsUtils from './ViolationsUtils';
```

Comment:
Why do we need this file?

Reply:
Yeah I think that's better, and that makes sense. Pardon my short review comments, I'm just trying to be concise but this was a genuine question, and not criticism

Reply:
No worries! I'm still figuring things out so I just wanted to declare my ignorance. 😅